frontend/src/HikeFinder.js

```diff
@@ -135,30 +127,39 @@ function HikeFinder(props) {
     }
   }
 
-  function updateInput(input) {
+  function updateBySearch(input) {
     console.log('here')
     const hikeData = props.hikeList.filter((_, i) =>
       finderState.filteredDataIndexes.has(i)
     )
-    let indices = new Set()
+    let indices = new Set([...finderState.filteredDataIndexes])
+    const beforeSearch = new Set([...finderState.filteredDataIndexes])
+    let searched = true;
     if (hikeData.length > 0) {
       for (let i = 0; i < hikeData.length; i++) {
-        if (hikeData[i].description.includes(input)) {
-          indices.add(i)
+        if (!(hikeData[i].description.includes(input))) {
+          indices.delete(i)
         }
       }
     }
 
-    if (indices.size === 0) {
+    console.log(hikeData)
+
+    if (input.length === 0 && finderState.recentlySearched) {
+      indices = new Set([...finderState.beforeSearchIndices])
+      searched = false;
+    }
+
+    else if (indices.size === 0) {
       console.log("Nothing matched your search results, please try something else!")
```

Comment:
> console.log("Nothing matched your search results, please try something else!")

This is only logged in the browser?

Reply:
Hey Bat! I definitely will change this. This is my first solution to responding to the user if a search result doesn't find anything. In the future, I am hoping to have a response on the frontend like a pop-up or just a text response under the search bar.